lib/macho/macho_file.rb

```diff
@@ -424,14 +478,24 @@ def low_fileoff
       offset
     end
 
+    # Updates the number of load commands in the raw data.
+    # @param ncmds [Fixnum] the new number of commands
+    # @return [void]
+    # @private
+    def set_ncmds(ncmds)
```

Comment:
Either as-is or `update_ncmds` would be fine by me. In the future, we might want to make this synchronize the `header` as well, to make its behavior self-contained (in which case `ncmds=` would be reasonable).

> Actually, let's just leave it as-is and change this later

:+1: